src/ethereum_spec_tools/evm_tools/t8n/__init__.py

```diff
@@ -62,6 +63,7 @@ def t8n_arguments(subparsers: argparse._SubParsersAction) -> None:
         "--state.reward", dest="state_reward", type=int, default=0
     )
     # TODO: Add support for the following trace options
+    t8n_parser.add_argument("--trace", action="store_true")
     t8n_parser.add_argument(
         "--trace.memory", dest="trace_memory", type=bool, default=False
```

Comment:
These arguments would then also have `action=store_true`.

Reply:
I can change the way these flags are used. However, I need to point out that this current PR does not implement these functionalities. So I would create a separate issue to track the implementation of these.

Reply:
Sounds good, they're not currently used in execution-exec-specs, but we'd like to add them at some point. I was playing around with memory tracing (from geth) and it was very helpful.